### PR TITLE
Add compat data for FileSystemEntry

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/FileSystemEntry

### CODE REVIEW

File: api/FileSystemEntry.json [hunk 1]
```diff
@@ -0,0 +1,618 @@
...
+          "standard_track": false,
+          "deprecated": true
+        }
+      },
+      "filesystem": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileSystemEntry/filesystem",
+          "support": {
+            "webview_android": {
+              "version_added": "0.16",
```
Comment (connorshea) [inline api/FileSystemEntry.json]: Is this actually a valid version?

**Comment by ddbeck** [comment]
@Elchi3 did you mean for me to review this or to fix it up and let someone else review it, like I've done with Mark's other outstanding PRs?

**Comment by Elchi3** [comment]
@ddbeck I was looking where we are with all File-related PRs and this one is still open. So, if you could do the fix-ups like you've done for the other File API PRs, it would be very much appreciated :) I probably should have set you as an assignee instead. Sorry for the confusion!

**Comment by ddbeck** [comment]
@Elchi3 No worries, thanks for the clarification. I'm aiming to get to this and the other PR you asked me to review done today :-)

File: api/FileSystemEntry.json (hunk 1 above)
Comment (ddbeck) [reply api/FileSystemEntry.json]: No it's not, I'll clean that up

**Pull request merged**